robolectric-resources/src/main/java/org/robolectric/res/ResName.java

```diff
@@ -32,7 +32,7 @@ public ResName(@NotNull String fullyQualifiedName) {
       throw new IllegalStateException("\"" + fullyQualifiedName + "\" is not fully qualified");
     }
     packageName = matcher.group(NAMESPACE);
-    type = matcher.group(TYPE);
```

Comment:
Couldn't this constructor just call 

this(packageName, type, nameStr)

and save the duplication?